Hadrons/A2AMatrix.hpp

```diff
@@ -104,11 +111,23 @@ class A2AMatrixIo
     void initFile(const MetadataType &d, const unsigned int chunkSize);
     // block I/O
     void saveBlock(const T *data, const unsigned int i, const unsigned int j,
-                   const unsigned int blockSizei, const unsigned int blockSizej);
+                   const unsigned int blockSizei, const unsigned int blockSizej, std::string datasetName="");
     void saveBlock(const A2AMatrixSet<T> &m, const unsigned int ext, const unsigned int str,
                    const unsigned int i, const unsigned int j);
+    //distillation overloads and new methods
+    // void saveStringMetadata(std::string name, const std::string value);
```

Comment:
in general, remove commented code unless it fits a specific documentation purpose

Reply:
Removed line 118 in commit d922033